api/openapi-spec/conventions-server..yaml

```
@@ -0,0 +1,133 @@
+openapi: 3.0.0
+info:
+  title: Conventions server 
+  description: >-
+    "a sample conventions server"
+  version: 1.0.0
+  license:
+    name: Apache-2.0
+    url: 'https://www.apache.org/licenses/LICENSE-2.0.html'
+paths:
+  /webhook:
+    post:
+      description: "a sample conventions server"
+      responses:
+        200:
+          description: Successfully applied defined conventions
+          content:
+            "application/json":
+              schema:
+                $ref: '#/components/schemas/PodConventionContext'
+        400:
+          description: Error decoding PodConventionContext from the request body or request body is nil 
+        500:
+          description: Internal server error
+          content:
+            "application/json":
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'
+components:
+  schemas:
+    PodConventionContext: 
+      type: object
+      properties:
+        apiVersions: 
+          type: string
+        kind: 
+          type: string
+        metadata:
+          type: array 
+          items: 
+            type: string
+        spec:
+          $ref: "#/components/schemas/PodConventionContextSpec"
+        status:
+          $ref: "#/components/schemas/PodConventionContextStatus"
+    PodConventionContextSpec:
+      type: object
+      properties:
+        template:
+          $ref: "#/components/schemas/PodTemplateSpec"
+        imageConfig:
+          type: array 
+          description: an array of imageConfig objects
+          items:
+            $ref: "#/components/schemas/ImageConfig"
+    PodTemplateSpec:
+      type: object 
+      properties:
+        spec: 
+          type: object
+          description: specification of the desired behavior of a pod
+        metadata:
+          type: object
+          properties:
+            annotations:
+              type: object
+              additionalProperties:
+                type: string
+            labels:
+              type: object
+              additionalProperties:
+                type: string
+    ImageConfig:
+      type: object 
+      properties: 
+        image: 
+          type: string 
+          description: a string reference to the image name and tag or associated digest
+          example: "example.com/repository/nginx:alpine"
+        BOMs:
+          type: array
+          description: "an array of bill of matearials (BOMs)"
+          items:
+            $ref: "#/components/schemas/BOM"
+        config: 
+          type: object 
+          description: "a ggcrv1 config file object"
+    BOM: 
+      type: object 
+      properties:
+        name: 
+          type: string 
+        raw: 
+          description: an array of bytes 
+          type: array
+          items:
+            type: object
```

the type in go is `[]byte` but on the wire it's a base64 encoded string.

```suggestion
          description: base64 encoded bytes 
          type: string
```